Patches/Vanilla Apparel Expanded/ThingDefs_Misc/Apparel_Industrial.xml

```diff
@@ -1,121 +1,176 @@
-				</value>
-			</li>
+				<!-- == VAE_Apparel_DoctorScrubs == -->
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_DoctorScrubs"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>30</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_LabCoat == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
-				<value>
-					<Bulk>7.5</Bulk>
-					<WornBulk>1</WornBulk>
-					<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_LabCoat == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<Bulk>7.5</Bulk>
+						<WornBulk>1</WornBulk>
+						<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
+					</value>
+				</li>
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>40</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_SheriffShirt == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
-				</value>
-			</li>
-		</operations>
+				<!-- == VAE_Apparel_SheriffShirt == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
+					</value>
+				</li>
+
+				<!-- == costStuffCounts == -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_TankTop"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>15</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_CasualTShirt" or defName="VAE_Apparel_Shorts" or defName="VAE_Apparel_Skirt" or defName="VAE_Apparel_Trousers"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>20</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_ShirtandTie" or defName="VAE_Apparel_Overalls" or defName="VAE_Apparel_FleeceShirt" or defName="VAE_Apparel_SheriffShirt"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>25</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_Hoodie" or defName="VAE_Apparel_Jeans" or defName="VAE_Apparel_ChefsUniform" or defName="VAE_Apparel_MilitaryUniform"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>30</costStuffCount>
```

Comment:
I think the military uniform should be down at least one more "tier" in regards to cost, because if we're using most real world military uniforms for reference, it consists of a pair of long pants, a long sleeve jacket, and a short sleeved undershirt--all fairly rugged and heavy-duty. I could easily see it costing 50 - 80 material.